.github/workflows/ci.yml

```diff
@@ -275,6 +275,10 @@ jobs:
         # javadoc task fails sporadically fetching https://docs.oracle.com/javase/8/docs/api/
         run: ./gradlew publishToMavenLocal -x javadoc
 
+      - name: Local publish gradle plugin
+        run: ../gradlew publishToMavenLocal -x javadoc
+        working-directory: gradle-plugins
+
```

Comment:
I think this won't be needed now that I finally got #4248 working

Reply:
👍 going to merge this PR since #4248 is getting reworked, and I'll take care of it in that PR